public/res_primaryLanguage.json

```diff
@@ -1,7 +1,7 @@
 {
     "basic_info": {
-      "description_header": "Hi",
-      "description": "👋 I'm Davina Griss. Fictional person for preview purposes :) I'm working with newest front-end frameworks like Angular, React and Vue. What you are seeing now is portfolio template from Dorota1997. If you like this portfolio template, make sure to ⭐ the repository to make it more recognizable for other users. Thank you 💜",
+      "description_header": "Creating Apps for caring life changers. By conscientious coding and aware communication.",
```

Comment:
I like the slogan as a general direction for now, but in production it will need to be refined for more clarity and pepp ;)